README.md

```diff
@@ -1,11 +1,51 @@
-# Xliff-tasks
-Set of tasks and targets to generate xliff files.
+# xliff-tasks
+
+A set of MSBuild tasks and targets to automatically generate xliff (.xlf) files for localizable resources, and to build satellite assemblies from those xliff files.
```

Comment:
Let's replace generate with synchronize. Since this is a common source of confusion.

Reply:
I think I'll go with "update".

Reply:
Sounds good. Matches the terminology in the code. :)